### PR TITLE
Fixed NPE in LoginUsernamePasswordFragment

### LINKED ISSUE #10832: EventBusException: Invoking subscriber failed
Sentry Issue: [WORDPRESS-ANDROID-1S](https://sentry.io/organizations/a8c/issues/1016040315/?referrer=github_integration)

```
NullPointerException: None
    at org.wordpress.android.ui.accounts.LoginActivity.loggedInAndFinish(LoginActivity.java:263)
    at org.wordpress.android.ui.accounts.LoginActivity.loggedInViaUsernamePassword(LoginActivity.java:603)
    at org.wordpress.android.login.LoginUsernamePasswordFragment.finishLogin(LoginUsernamePasswordFragment.java:466)
    at org.wordpress.android.login.LoginUsernamePasswordFragment.onSiteChanged(LoginUsernamePasswordFragment.java:518)
...
(11 additional frame(s) were not displayed)

EventBusException: Invoking subscriber failed
    at org.greenrobot.eventbus.EventBus.handleSubscriberException(EventBus.java:527)
    at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:509)
    at org.greenrobot.eventbus.EventBus.invokeSubscriber(EventBus.java:501)
    at org.greenrobot.eventbus.HandlerPoster.handleMessage(HandlerPoster.java:67)
    at android.os.Handler.dispatchMessage(Handler.java:106)
...
(5 additional frame(s) were not displayed)

Invoking subscriber failed
```

### PR DESCRIPTION
**Pull request by khaykov**
Fixes #10832 

After the user leaves `LoginUsernamePasswordFragment` for more than one hour and comes back, a global site refresh is triggered, which causes `onSiteChanged` in `LoginUsernamePasswordFragment` to propagate false "successful" login event, which causes a crash down the line.

To make the testing easier, change the throttle delay on-site refresh ([here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/WordPress.java#L133)) to something reasonable, like 10 seconds.

To test:
- Navigate to Site Picker and press the + icon to add new site.
- Chose "Add self-hosted site" option.
- In the Site address field input self-hosted site URL and press NEXT.
- While on the screen with login and password, press the home button and minimize the app.
- Wait 10 seconds (or whatever time you chouse for site fetch throttle.
- Navigate back to the app, and notice that it's not crashing.

Part 2:
- Input wrong login or password and press NEXT to produce an error.
- Press the home button and minimize the app.
- Navigate back to the app and input correct login credentials.
- Make sure you can login and the site is added.

PR submission checklist:

- [x] I have considered adding unit tests where possible.

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java [hunk 1]
```diff
@@ -253,6 +256,7 @@ public void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
 
         outState.putBoolean(KEY_LOGIN_FINISHED, mLoginFinished);
+        outState.putBoolean(KEY_LOGIN_FINISHED, mLoginStarted);
```
Comment (planarvoid) [inline libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java]: I think the key here is wrong, it should be `KEY_LOGIN_STARTED`

Comment (malinajirka) [reply libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java]: :D How come I've missed this. Great catch !

Comment (malinajirka) [reply libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java]: :D How come I've missed this. Great catch !

**Comment by jkmassel** [comment]
We're freezing `13.8` today, so I'm moving this forward to `13.9`. If you'd like it released as part of `13.8`, please adjust the target from `develop` to `release/13.8`, and let me know once this PR has been merged – I'll be happy to ship a new release candidate!

**Comment by khaykov** [comment]
I fixed the issue, so this one is ready for another round :)

**Pull request merged**